include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -295,7 +295,7 @@ namespace parallel
        * active cells and will be ordered by the occurrence of those cells.
        */
       virtual void
-      update_cell_relations() override;
+      update_cell_relations();
 
```

Comment:
Now that we don't overload the virtual method from the base class `dealii::Triangulation` any more (as you removed the `virtual` there), does this still have to be `virtual`. Could you quickly explain your reasoning here?

Reply:
You are right. They can be removed.